bundle-workflow/python/paths/script_finder.py

```diff
@@ -7,16 +7,22 @@
 @dataclass
 class ScriptFinder:
     '''
-    ScriptFinder is a helper that abstracts away the details of where to look for build and test scripts.
-    Given a component name and a checked-out Git repository, it will look in the following locations, in order, for either "build.sh" or "integtest.sh", depending on the use case:
+    ScriptFinder is a helper that abstracts away the details of where to look for build, test and install scripts.
+
+    For build.sh and integtest.sh scripts, given a component name and a checked-out Git repository, 
+    it will look in the following locations, in order:
       * Root of the Git repository
       * /scripts/<script-name> in the Git repository
       * <component_scripts_path>/<component_name>/<script-name>
       * <default_scripts_path>/<script-name>
+
+    For install.sh scripts, given a component name, it will look in the following locations, in order:
+      * <component_scripts_path>/<component_name>/<script-name>
+      * <default_scripts_path>/<script-name>
     '''
 
-    component_scripts_path: str
-    default_scripts_path: str
+    component_scripts_path: str = os.path.join(os.path.dirname(os.path.abspath(__file__)), '../../scripts/components')
+    default_scripts_path: str = os.path.join(os.path.dirname(os.path.abspath(__file__)), '../../scripts/default')
```

Comment:
Relying on `__file__` and relative paths is going to be fragile...this is OK for now, but consider using a regular constructor that accepts an optional base path instead of using `@dataclass`. dataclass is not really appropriate for this class anyway - it's not a plain data object and it has no fields that should be readable by clients.

Reply:
I actually think this is going to equally fragile than specifying it 3 times in every calling code :)

Reply:
So lets get rid of at least one of those pieces of fragility :)